code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -322,7 +322,8 @@
 				if(HAS_TRAIT(src, TRAIT_DUMB))
 					msg += "[t_He] [t_has] a stupid expression on [t_his] face.\n"
 		if(get_organ_by_type(/obj/item/organ/internal/brain))
-			if(!key)
+			if (ai_controller)
```

Comment:
`if(get_organ_by_type(/obj/item/organ/internal/brain) && isnull(ai_controller))`

Reply:
I had it this way with the thought we may possibly add lines to the examine_text specifically for AI controlled mobs in the future. Should I comment that in, or implement this suggestion instead and just leave refactor for that future possibility?

Reply:
If someone wants to add lines to AI examines they can separate it out of this line themselves.